examples/simple/README.md

```diff
@@ -0,0 +1,6 @@
+
+# Simple Trace Example
+
+In this example, an application in `main.cc` initializes and registers a tracer provider from the <a href="https://github.com/open-telemetry/opentelemetry-cpp"> Open Telemetry SDK </a>.  The application then calls a `foo_library` which has been manually instrumented using the <a href="https://github.com/open-telemetry/opentelemetry-cpp/tree/master/api"> OpenTelemetry API </a>.  Resulting telemetry is directed to stdout through a custom exporter.
```

Comment:
```suggestion
In this example, an application in `main.cc` initializes and registers a tracer provider from the <a href="https://github.com/open-telemetry/opentelemetry-cpp"> Open Telemetry SDK </a>.  The application then calls a `foo_library` which has been instrumented using the <a href="https://github.com/open-telemetry/opentelemetry-cpp/tree/master/api"> OpenTelemetry API </a>.  Resulting telemetry data is directed to stdout through a custom exporter.
```

Reply:
_manually instrumented_ doesn't really make sense, as we only support manual instrumentation. This might suggest we can do automatic instrumentation, but we don't.